<?xml version="1.0"?>
<launch>

	<!-- Goal Allocation  - Simulated Annealing -->
	<node pkg="demeter_planning" type="goal_allocation_sa.py" name="goal_allocation" output="screen"> 
		<rosparam file="$(find demeter_planning)/params/vehicles_in_the_system.yaml" command="load"/>
		<rosparam file="$(find demeter_planning)/params/scaled_turbines_xy.yaml" command="load"/>
		<rosparam file="$(find demeter_planning)/params/waypoints.yaml" command="load"/>
	</node>

</launch>